dev-qt/libqt4pas/libqt4pas-2.5.0.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Pascal Qt interface"
+DESCRIPTION="Pascal wrapper around Qt interfaces.
+Just the necessary stuff for Lazarus is wrapped here."
+HOMEPAGE="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/fpcqt4.html"
+COPYRIGHT="2005-2011 Jan Van hijfte"
+LICENSE="GNU LGPL v2.1"
+REVISION="1"
+SOURCE_URI="http://users.telenet.be/Jan.Van.hijfte/qtforfpc/V2.5/qt4pas-V2.5_Qt4.5.3.tar.gz"
+CHECKSUM_SHA256="825423db80da4df5c21816c0392b3394cddfe2f3293dfd08ace84941726affea"
+SOURCE_DIR="qt4pas-V2.5_Qt4.5.3"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	libqt4pas$secondaryArchSuffix = $portVersion
+	lib:libqt4pas$secondaryArchSuffix = 5.2.5 compat >= 5.2
+	"
+REQUIRES="
+	haiku${secondaryArchSuffix}
+	lib:libqtcore$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	libqt4pas$secondaryArchSuffix = $portVersion
+	#lib:libQt4Pas$secondaryArchSuffix = $portVersion compat >= 2.2
+	devel:libqt4pas$secondaryArchSuffix = 5.2.5 compat >= 5.2
+	"
+REQUIRES_devel="
+	haiku$secondaryArchSuffix
+	lib:libqtcore${secondaryArchSuffix}
+	devel:libqtcore${secondaryArchSuffix}
+	cmd:qmake$secondaryArchSuffix
+	cmd:qmake
+	cmd:make
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	cmd:make
+	cmd:qmake$secondaryArchSuffix
+	cmd:g++$secondaryArchSuffix
+	lib:libqtcore$secondaryArchSuffix
+	devel:libqtcore$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:gcc${secondaryArchSuffix}
+	cmd:ld${secondaryArchSuffix}
+	lib:libqtcore$secondaryArchSuffix
+	devel:libqtcore$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	qmake 
+	make 
+}
+
+INSTALL()
+{
+	make install
+	mkdir -p $libDir
+	cp libQt4Pas.so* $libDir
+	# prepare develop/lib
+	prepareInstalledDevelLibs libQt4Pas
```

Comment:
Since there is only one lib, you can use prepareInstalledDevel**Lib** instead of  prepareInstalledDevel**Libs**:
```
	prepareInstalledDevelLib libQt4Pas
```